docs/infrastructure/requesting-infrastructure.md

```diff
@@ -0,0 +1,59 @@
+# Requesting infrastructure
+
+This page details the process for requesting new infrastructure. This aims to be
+as lightweight as possible, while capturing all the information that's needed.
+
+Unless otherwise noted, the way to request things is to contact the
+Infrastructure Team by starting a new conversation thread in the
+[`#infrastructure` channel][slack-infra] in Slack. Please ping
+`@infrastructure-committee` to ensure that your message is seen.
+
+In general you can expect a response within a couple of days (often much
+sooner), though note that the team may not always be able to act on your request
+for some time.
+
+Requests for new infrastructure must be made by (or confirmed by) members of a
+Team Committee.
+
+Where infrastructure requests would incur additional spending (for example a new
+machine from Mythic Beasts), you must include a reference to the line in the
+Team's budget to which that spending would relate.
+
+[slack-infra]: https://studentrobotics.slack.com/messages/infrastructure
+
+## GitHub
+
+When requesting changes related to GitHub, please include the GitHub username of
+the users affected as well as the names of any teams or repos those people will
```

Comment:
> the users affected

What if it's a (github) team?

Reply:
How about:
> When requesting changes related to GitHub users, please include the GitHub
> username of the users affected as well as the names of any teams or repos those
> people will need access to.

I'm not sure how to frame the teams thing without it just sounding like wasted words. Open to suggestions.

Reply:
f4b8979